taca/storage/storage.py

```diff
@@ -69,15 +66,12 @@ def cleanup_processing(days):
                 for run in [r for r in os.listdir(archive_dir) if re.match(filesystem.RUN_RE, r)]:
                     rta_file = os.path.join(run, finished_run_indicator)
                     if os.path.exists(rta_file):
-                        # 1 day == 60*60*24 seconds --> 86400
-                        if os.stat(rta_file).st_mtime < time.time() - (86400 * days) and \
-                                filesystem.is_in_swestore("{}.tar.bz2".format(run)):
-                            logger.info('Removing run {} to nosync directory'
-                                        .format(os.path.basename(run)))
+                        if os.stat(rta_file).st_mtime < time.time() - seconds:
```

Comment:
yes, as we are not using (and not going to use) `SWESTORE` anymore, I believe it is not necessary(?)